library/swift/src/GRPCStreamEmitter.swift

```diff
@@ -43,7 +43,9 @@ public final class GRPCStreamEmitter: NSObject {
   }
 
   /// Close this connection.
-  public func close() {
+  public func close(_ data: Data) {
+    // The gRPC protocol requires the client stream to close with a DATA frame.
+    // More information here: https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md#requests
     self.underlyingEmitter.close()
```

Comment:
This is actually wrong, it'll still send trailers here rather than empty data.

Reply:
How come? I thought by deleting the extension below, I was getting rid of a default value when no argument was given

Reply:
This comment was before the extension was deleted. I don't think this compiles as-is, you'll need to specify `trailers: nil` I believe

Reply:
😂 I think my commit and your comment cross paths. Will update